main.go

```diff
@@ -247,11 +238,7 @@ func sendToWriter(ingestionType string, msgBody *strings.Reader, uuid string, ur
 	request, err := http.NewRequest("PUT", reqURL, msgBody)
 	request.ContentLength = -1
 
-	client1 := pester.New()
-	client1.MaxRetries = 3
-	client1.Backoff = pester.ExponentialBackoff
-
-	resp, err := client1.Do(request)
+	resp, err := pester.DefaultClient.Do(request)
```

Comment:
I see this client has a Concurrency setting of 1 - does that mean it can only be used by 1 subroutine at a time? If yes, is that what we want?